src/GirCore.sln

```diff
@@ -283,6 +284,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "scripts", "scripts", "{FF81
 		..\scripts\GenerateLibs.fsx = ..\scripts\GenerateLibs.fsx
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Secret-1", "Libs\Secret-1\Secret-1.csproj", "{56D500D0-814A-49D6-A56F-813E2285C8F1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Secret-1.Tests", "Tests\Libs\Secret-1.Tests\Secret-1.Tests.csproj", "{DF75306B-D705-4853-AFA6-20D04EC22AAE}"
```

Comment:
Please remove the test project completely. I don't test if individual library code is working like intended. I just verify if the generated code works like expected vial the "GirTest" project.

There are some exceptions because there are some very fundamental things I like to test or sometimes there are just tests which were not yet converted from e.g. `Gtk-4.0.Tests` to `GirTest-0.1.Tests`.